Tasks/Xcode/task.json

```diff
@@ -226,6 +231,55 @@
             "visibleRule": "signStyle = auto"
         },
         {
+            "name": "platform",
+            "type": "picklist",
+            "label": "Destination platform",
+            "defaultValue": "default",
+            "required": false,
+            "helpMarkDown": "Select the destination device's platform. If you need an option not listed below, use `Arguments` to pass the -destination argument.",
+            "groupName": "devices",
+            "options": {
```

Comment:
We can have another option "custom" and let use enter the platform.

Reply:
Maybe on-prem tips the balance to where we should add custom. I'm a little worried about whether the new platform will follow the same pattern as iOS and tvOS. The macOS platform takes other arguments. You can't pass a `name=` or `id=` with macOS.

  -destination "platform=macOS;arch=i386"

Reply:
Done. I named the new input just `destinationPlatform` so you can do this in yaml:

 - destinationPlatform: "fooOS"

Instead of:

 -destinationPlatformOption: "custom"
 -destinationPlatformCustom: "fooOS"